README.md

```diff
@@ -0,0 +1,87 @@
+
+# Slurm charms
+
+[Juju](https://juju.is) charms for automating the Day 0 to Day 2 operations of the [Slurm workload manager](https://slurm.schedmd.com/overview.html) ⚖️🐧
+
+[![CI](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/ci.yaml/badge.svg)
+[![Release](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)](https://github.com/charmed-hpc/slurm-charms/actions/workflows/release.yaml/badge.svg)
+[![Matrix](https://img.shields.io/matrix/ubuntu-hpc%3Amatrix.org?logo=matrix&label=ubuntu-hpc)](https://matrix.to/#/#hpc:ubuntu.com)
+
+The `slurm-charms` repository provides many of the Slurm daemons that are required to deploy, configure and manage a Slurm-based HPC cluster:
+
+- [`slurmctld`](./charms/slurmctld/): the central management daemon for Slurm.
+- [`slurmd`](./charms/slurmd): the compute node daemon for Slurm.
+- [`slurmdbd`](./charms/slurmdbd): the database daemon for Slurm.
+- [`slurmrestd`](./charms/slurmrestd/): the REST API interface to Slurm.
+
+## ✨ Getting started
+
+This should be run using Juju 3.x or greater.
+
+```shell
+$ juju deploy slurmctld --channel edge
+$ juju deploy slurmd --channel edge
+$ juju deploy slurmdbd --channel edge
+$ juju deploy slurmrestd --channel edge
+$ juju deploy mysql --channel 8.0/stable
+$ juju deploy mysql-router slurmdbd-mysql-router --channel dpe/edge
+
+$ juju integrate slurmctld:slurmd slurmd:slurmctld
+$ juju integrate slurmctld:slurmdbd slurmdbd:slurmctld
+$ juju integrate slurmctld:slurmrestd slurmrestd:slurmctld
+$ juju integrate slurmdbd-mysql-router:backend-database mysql:database
+$ juju integrate slurmdbd:database slurmdbd-mysql-router:database
+```
+
+## 🤔 What's next?
+
+If you want to know more about the project, there are several links available for you to explore:
+
+* [Documentation](https://canonical-charmed-hpc.readthedocs-hosted.com/en/latest)
+* [Open an issue](https://github.com/charmed-hpc/slurm-charms/issues/new?title=ISSUE+TITLE&body=*Please+describe+your+issue*)
+* [Ask a question on Github](https://github.com/orgs/charmed-hpc/discussions/categories/q-a)
+
+## 🛠️ Development
+
+The project uses [tox](tox.wiki) as its command runner. It's very common to run the following four
+commands to check that the codebase is in a good state.
+
+```shell
+tox run -e fmt # Apply correct formatting to code.
```

Comment:
I'm not sure if "correct formatting" is the right phrasing here? Wdyt about "formatting standards" instead?

```suggestion
tox run -e fmt # Apply formatting standards to code.
```

Also, nit, but I really want the comments to line up together for some reason :rofl:

Reply:
I'll also change tox's job name because this is just copied from there.